### PR TITLE
MB - Task 6 and 7

### PR DESCRIPTION
**Pull request by MatiasBonifacio**
I've made the quality gates run and add the docker image to my account
jenkins
docker image

### CODE REVIEW

File: MB-Jenkins.gocd.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  Frame-Extractor:
+    group: latam02
+    label_template: ${COUNT}
+    lock_behavior: none
+    display_order: -1
+    environments_variables:
+      IMAGE_NAME: 'Frame_Extracor'
+      TAG_VERSION: '1.0'
+    secure_variables:
+      SONAR_TOKEN: 'sonar_token'
```
Comment (vcarmen) [inline MB-Jenkins.gocd.yml:12]: @MatiasBonifacio , where is the encrypted value for the sonar token?

File: MB-Jenkins.gocd.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+format_version: 10
```
Comment (vcarmen) [inline MB-Jenkins.gocd.yml:1]: Rename the file to be build.gocd.yml, this is not Jenkins

File: Jenkinsfile [hunk 3]
```diff
@@ -28,31 +32,35 @@ pipeline {
...
+    stage('Publish'){
+       steps {
+         sh 'docker login -u ${DOCKER_USER} -p ${DOCKER_PASSWORD}'
+         sh 'docker tag ${IMAGE_NAME}:${TAG_VERSION} ${DOCKER_USER}/${IMAGE_NAME}:${TAG_VERSION}'
+         sh 'docker push ${DOCKER_USER}/${IMAGE_NAME}:${TAG_VERSION}'
+       }
+     }
+    stage('Deploy'){
+       steps {
+         sh 'echo deploy'
```
Comment (vcarmen) [inline Jenkinsfile:61]: You need to include the deployment docker-compose command here

Comment (vcarmen) [reply Jenkinsfile:61]: Remember the docker-compose file uses the image you just published step above.

**Review by vcarmen — CHANGES_REQUESTED** [review]

File: MB-Jenkins.gocd.yml (hunk 1 above)
Comment (MatiasBonifacio) [reply MB-Jenkins.gocd.yml:12]: That is for task 8. i've created another branch to finish that.

**Pull request closed without merging**